crates/ntx-builder/src/block_producer.rs

```diff
@@ -49,7 +49,10 @@ impl BlockProducerClient {
         &self,
         proven_tx: ProvenTransaction,
     ) -> Result<(), Status> {
-        let request = proto::transaction::ProvenTransaction { transaction: proven_tx.to_bytes() };
+        let request = proto::transaction::ProvenTransaction {
+            transaction: proven_tx.to_bytes(),
+            transaction_inputs: None,
+        };
```

Comment:
This would mean that the validator would have no knowledge of the transactions created by the NTB, right? This is fine for now, but once we implement block signing, this will become a problem. So, we'll need to figure out how does NTB communicate its transactions to the validator.

Reply:
OK I have made an issue for NTB followup https://github.com/0xMiden/miden-node/issues/1309